front/src/modules/trainschedule/components/Timetable/PacedTrain/PacedTrainItem.tsx

```diff
@@ -55,9 +56,10 @@ const PacedTrainItem = ({
   isOnEdit,
   isProjectionPathUsed,
   selectPacedTrainToEdit,
-  selectedTrainId,
+  selectedTimetableItemId,
```

Comment:
We select a train (occurence or trainSchedule), not a timetableItem (pacedTrain or trainSchedule)
```suggestion
  selectedTrainId,
```

Reply:
done